includes/ppc-frontend.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * Pre-Publish Checklist php for Displaying the contents in the general tab in the plugin settings
+ * PHP version 7
+ *
+ * @category PHP
+ * @package  Pre-Publish Checklist.
+ * @author   Display Name <username@ShubhamW.com>
+ * @license  http://brainstormforce.com
+ * @link     http://brainstormforce.com
+ */
+
+// Displaying the contents in the general tab in the plugin settings.
+$ppc_radio_button        = get_option( 'ppc_radio_button_option_data' );
```

Comment:
The option name `ppc_radio_button_option_data` should be more relevant to what this option is about.
Also, change this anywhere else this is relevant, for eg in the localized variable and in JS.

Example name - `ppc_error_level`.

Also, you can pass `3` as the third parameter for default value instead of setting the default value on the next line.